server/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
package.json is setup now, so you should be able to pull this code and run "npm install" to get all the dependencies you need rather than doing a "git init" first.